config/locales/server.en.yml

```diff
@@ -1092,6 +1092,7 @@ en:
     poll_pop3_timeout: "Connection to the POP3 server is timing out. Incoming email could not be retrieved. Please check your <a href='/admin/site_settings/category/email'>POP3 settings</a> and service provider."
     poll_pop3_auth_error: "Connection to the POP3 server is failing with an authentication error. Please check your <a href='/admin/site_settings/category/email'>POP3 settings</a>."
     force_https_warning: "Your website is using SSL. But `<a href='/admin/site_settings/category/all_results?filter=force_https'>force_https</a>` is not yet enabled in your site settings."
+    out_of_date_themes: "Updates are available for the following themes. Please update them!"
```

Comment:
Updates are available for the following themes.

The "Please update them!" is a bit pushy, and potentially it can break stuff so people need to know about it but not rush.

Reply:
Ok I've removed that bit.